src/System.ComponentModel.Primitives/src/System/ComponentModel/Component.cs

```diff
@@ -0,0 +1,257 @@
+//------------------------------------------------------------------------------
+// <copyright file="Component.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>                                                                
+//------------------------------------------------------------------------------
```

Comment:
I rand the codeformatter on the TypeConverter project, but I missed the files I added to the Primitives folder.